src/dosbox.cpp

```diff
@@ -666,15 +666,18 @@ void DOSBOX_Init()
 	pstring->Set_help("CPU core used in emulation ('auto' by default). 'auto' will switch to dynamic\n"
 	                  "if available and appropriate.");
 
-	const char* cputype_values[] = { "auto", "386", "386_slow", "486_slow", "pentium_slow",
-#if C_MMX
-	                                  "pentium_mmx",
-#endif
-	                                  "386_prefetch", nullptr};
+	const char* cputype_values[] = { "auto", "386", "486", "pentium",
+	                                 "pentium_mmx", "386_prefetch",
+	                                 nullptr };
```

Comment:
Ok, so let's go with this order and let's expose the old `386` setting a `386_fast`. We don't quite understand what we'd lose by removing it, so let's keep it for now.

```
"auto", "386", "386_fast", "386_prefetch", "486", "pentium", "pentium_mmx"
```

Reply:
We don't want to overcomplicate it so let's stay with my above suggestion. Maybe we'll mention these details in the manual, but these detail say very little to 99.99% of users.

Reply:
OK, changed. I have tried to benchmark "386_fast" against "386", but - strangely - I'm getting same results.